site/config.yaml

```diff
@@ -6,103 +6,104 @@ DefaultContentLanguage: "en"
 title: "Carvel"
 theme: "carvel"
 frontmatter:
-  date: [":filename", ":default"]
+ date: [":filename", ":default"]
 params:
-  products: [ytt, kbld, kapp, imgpkg, kapp-controller, vendir]
-  ytt:
-    name: ytt
-    root_link: /ytt/
-    latest_docs_link: /ytt/docs/latest/
-    github_url: https://github.com/vmware-tanzu/carvel-ytt
-    search: true
-    search_index_name: carvel
-    search_api_key: 582b1408d7a3fe145c0f84f48ca03755
-    search_tag: ytt
-    versioning: false
-    version_latest: latest
-    versions:
-      - latest
-  kbld:
-    name: kbld
-    root_link: /kbld/
-    latest_docs_link: /kbld/docs/latest/
-    github_url: https://github.com/vmware-tanzu/carvel-kbld
-    search: true
-    search_index_name: carvel
-    search_api_key: 582b1408d7a3fe145c0f84f48ca03755
-    search_tag: kbld
-    versioning: false
-    version_latest: latest
-    versions:
-      - latest
-  kapp:
-    name: kapp
-    root_link: /kapp/
-    latest_docs_link: /kapp/docs/latest/
-    github_url: https://github.com/vmware-tanzu/carvel-kapp
-    search: true
-    search_index_name: carvel
-    search_api_key: 582b1408d7a3fe145c0f84f48ca03755
-    search_tag: kapp
-    versioning: false
-    version_latest: latest
-    versions:
-      - latest
-  imgpkg:
-    name: imgpkg
-    root_link: /imgpkg/
-    latest_docs_link: /imgpkg/docs/latest/
-    github_url: https://github.com/vmware-tanzu/carvel-imgpkg
-    search: true
-    search_index_name: carvel
-    search_api_key: 582b1408d7a3fe145c0f84f48ca03755
-    search_tag: imgpkg
-    versioning: false
-    version_latest: latest
-    versions:
-      - latest
-  kapp-controller:
-    name: kapp-controller
-    root_link: /kapp-controller/
-    latest_docs_link: /kapp-controller/docs/latest/
-    github_url: https://github.com/vmware-tanzu/carvel-kapp-controller
-    search: true
-    search_index_name: carvel
-    search_api_key: 582b1408d7a3fe145c0f84f48ca03755
-    search_tag: kapp-controller
-    versioning: false
-    version_latest: latest
-    versions:
-      - latest
-  vendir:
-    name: vendir
-    root_link: /vendir/
-    latest_docs_link: /vendir/docs/latest/
-    github_url: https://github.com/vmware-tanzu/carvel-vendir
-    search: true
-    search_index_name: carvel
-    search_api_key: 582b1408d7a3fe145c0f84f48ca03755
-    search_tag: vendir
-    versioning: false
-    version_latest: latest
-    versions:
-      - latest
-  links:
-    twitter_url: https://twitter.com/carvel_dev
-    github_url: https://github.com/vmware-tanzu/carvel.dev
-    slack_url: https://kubernetes.slack.com/archives/CH8KCCKA5
```

Comment:
Generally, our YAML indentation convention is two spaces. I'm curious why this was changed to one space?